### PR TITLE
OPENEUROPA-1322: Use the proper Docker way of overriding docker-composer.yml file. 

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@
 /phpunit.xml
 /runner.yml
 /vendor
+docker-compose.*.yml
```
Comment (dxvargas) [inline .gitignore]: It's missing a new line in the end.

File: composer.json [hunk 2]
```diff
@@ -7,18 +7,17 @@
...
     },
     "require-dev": {
         "composer/installers": "^1.2",
-        "cweagans/composer-patches": "~1.0",
         "drupal-composer/drupal-scaffold": "^2.5",
         "drush/drush": "^9.3",
         "openeuropa/code-review": "^0.3",
         "openeuropa/task-runner": "^0.7.0",
-        "webflo/drupal-core-require-dev": "8.5.x"
+        "openeuropa/drupal-core-require-dev": "8.6.x-dev"
```
Comment (dxvargas) [inline composer.json]: It should be the same as "openeuropa/drupal-core-require": "^8.6".
It should be in alphabetic order.

File: composer.json [hunk 3]
```diff
@@ -7,18 +7,17 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.1",
-        "drupal/core": "~8.5",
+        "openeuropa/drupal-core-require": "^8.6",
```
Comment (dxvargas) [inline composer.json]: It should be in alphabetic order.

File: .gitignore (hunk 1 above)
Comment (drupol) [reply .gitignore]: Fixed.

File: composer.json (hunk 2 above)
Comment (drupol) [reply composer.json]: Fixed.

File: composer.json (hunk 3 above)
Comment (drupol) [reply composer.json]: Fixed.

File: README.md [hunk 4]
```diff
@@ -62,7 +75,7 @@ database host, and change your base URL and database credentials.
 *Step 3: Build*
 
 ```bash
-$ ./vendor/bin/run drupal:site-setup
+./vendor/bin/run drupal:site-setup
```
Comment (dxvargas) [inline README.md]: It should be removed like in other repos. Not a blocker, though.

**Pull request merged**